spec/lib/open_food_network/scope_variants_to_search_spec.rb

```diff
@@ -30,8 +30,8 @@
       let(:params) { { q: "product 1" } }
 
       it "returns all products whose names or SKUs match the query" do
-        expect(result).to include v1, v2
-        expect(result).to_not include v3, v4
+        expect(result).to include v1
+        expect(result).to_not include v2, v3, v4
```

Comment:
This was an odd one because the actual code in the variant search was only really looking at variant SKUs and not product SKUs and that seemed reasonable. I took another look and I've updated it to match on both of them now here: https://github.com/openfoodfoundation/openfoodnetwork/pull/10939/commits/cda903c51168f8c10a9d67ab0864ac9a32fe412e

Reply:
Bravo!